framework/src/meshgenerators/AnnularMeshGenerator.C

```diff
@@ -62,27 +74,37 @@ AnnularMeshGenerator::AnnularMeshGenerator(const InputParameters & parameters)
     _nt(getParam<unsigned int>("nt")),
     _rmin(getParam<Real>("rmin")),
     _rmax(getParam<Real>("rmax")),
-    _tmin(getParam<Real>("tmin")),
-    _tmax(getParam<Real>("tmax")),
+    _dmin(parameters.isParamSetByUser("tmin") ? getParam<Real>("tmin") / M_PI * 180.0
+                                              : getParam<Real>("dmin")),
+    _dmax(parameters.isParamSetByUser("tmax") ? getParam<Real>("tmax") / M_PI * 180.0
+                                              : getParam<Real>("dmax")),
+    _radians((parameters.isParamSetByUser("tmin") || parameters.isParamSetByUser("tmax")) ? true
+                                                                                          : false),
     _growth_r(getParam<Real>("growth_r")),
     _len(_growth_r == 1.0 ? (_rmax - _rmin) / _nr
                           : (_rmax - _rmin) * (1.0 - _growth_r) / (1.0 - std::pow(_growth_r, _nr))),
-    _full_annulus(_tmin == 0.0 && _tmax == 2 * M_PI),
+    _full_annulus(_dmin == 0.0 && _dmax == 360),
     _quad_subdomain_id(getParam<SubdomainID>("quad_subdomain_id")),
     _tri_subdomain_id(getParam<SubdomainID>("tri_subdomain_id"))
 {
-  // catch likely user errors
+  if ((parameters.isParamSetByUser("tmin") || parameters.isParamSetByUser("tmax")) &&
+      (parameters.isParamSetByUser("dmin") || parameters.isParamSetByUser("dmax")))
+    paramError("tmin",
+               "Both Ddegree and radian specifications are provided. Please use degree "
```

Comment:
Same comment as before

Reply:
OK, the "before" comment didn't appear - i must have not clicked correctly.  It was
Ddegree -> degree
but perhaps a better sentence would be "You specified the angles using both degrees and radians."

Reply:
Do you mean the Ddegree typo?